src/notification/JoinNotification.js

```diff
@@ -0,0 +1,39 @@
+import { RichEmbed, Guild } from "discord.js";
+import GrouperClient from "../structures/GrouperClient";
+import Notification from './Notification.js'
+
+export default class JoinNotification extends Notification {
+    /**
+     * Creates a new notification handler for when guilds are joined
+     * 
+     * @param {GrouperClient} client 
+     * @param {String} guild - Support server snowflake
+     * @param {String} channel - Alert channel snowflake
+     */
+    constructor(client, guild, channel) {
+        super(client, guild, channel);
+    }
+
+    /**
+     * Builds an embed based off of the information passed into data
+     * 
+     * @param {Guild} data - Data
+     * 
+     * @return {RichEmbed}
+     */
+    buildEmbed(data) {
+        const embed = new RichEmbed()
```

Comment:
The declaration of the embed variable is redundant for now as you're just returning it straight away anyway.